repo_validation/scripts/validate_srs_consistency.ps1

```diff
@@ -86,24 +86,23 @@ function Test-IsExcluded {
 # Function to strip markdown formatting from text
 function Remove-MarkdownFormatting {
     param([string]$Text)
-    
-    # Remove bold/italics markers (**text**, *text*)
-    $Text = $Text -replace '\*\*([^\*]+)\*\*', '$1'
-    $Text = $Text -replace '\*([^\*]+)\*', '$1'
-    
+
+    # Remove bold/italics markers (**text**, *text*) - be more careful about nested formatting
+    # Handle nested bold like **unsigned char** properly
```

Comment:
> # Handle nested bold like **unsigned char** properly [](http://example.com/codeflow?start=3&length=53)

not sure what the "nested bold" is in the comment. Should the example be something like "**Nested bool  like **unsigned char** properly**"?

I assume this handles things like * literals in the middle?

E.g.

Shall *multiply 1 * 2*
becomes
Shall multiply 1 * 2

Or maybe more common would be a "*" used as a pointer.